src/dataflow/src/sink/kafka.rs

```diff
@@ -333,44 +341,27 @@ where
                     continue;
                 }
 
+                assert!(diff >= 0, "can't sink negative multiplicities");
```

Comment:
I think you can assert > 0 actually because everything has a diff of one before its consolidated and sent to this operator

Reply:
You're right, but I wanted to keep these operators as generic as possible and not rely on the invariant that consolidate is called upstream.